plugins/Identify.jsx

```diff
@@ -61,7 +66,18 @@ class Identify extends React.Component {
     state = {
         mode: 'Point',
         identifyResults: null,
-        pendingRequests: 0
+        pendingRequests: 0,
+        layer: null,
+        radius: 50,
+        units: 'meters'
+    }
+    constructor(props, context) {
+        super(props, context);
+        this.state.layer = {
+            id: "buffer",
+            title: LocaleUtils.tr("identify.bufferlayername"),
+            role: LayerRole.USERLAYER
+        };
```

Comment:
`buffer` is a bit too generic as ID, owuld call it `identifyradiusbuffer` or similar.

There is probably no point having this layer show in the layertree, so I'd set `role: LayerRole.SELECTION` (which are automatically omitted from the LayerTree) and omit the `title`.

I wouldn't store this in the component state, rather just as a local variable in `computeBuffer()`.